.vscode/settings.json

```diff
@@ -5,9 +5,19 @@
             "*.unsafe:rust": "#eb5046"
         }
     },
+    "files.autoGuessEncoding": true,
+    "files.exclude": {
+        "venv": true,
+        ".pixi": true,
+        "build": true,
+        "compare_screenshot": true,
+        "target": true,
+        "target_ra": true,
+        "target_wasm": true,
+        "web_demo": true
+    },
```

Comment:
Could it be that something is wrong with your user settings? My search & jump to file don't brings up anything that's on .gitignore

Reply:
Aha, that did the trick. Dropping that part of the change.